.github/workflows/ci.yml

```diff
@@ -14,19 +14,25 @@ jobs:
     outputs:
       script: ${{ steps.get-scripts.outputs.script }}
     steps:
-      - uses: actions/checkout@v2
-      - id: get-scripts
-        # it's weird to me, but the quotes around \n should *not* be escaped or it breaks
-        run: echo "::set-output name=script::$(ls tests/test*py | jq -R -s -c 'split("\n")[:-1]')"
+    - uses: actions/checkout@v3
+    - id: get-scripts
+        # it's weird to me, but the quotes around \n should *not* be escaped or
+        # it breaks. we cd into tests so that the output of this is just e.g.,
+        # "test_tools.py", not "tests/test_tools.py", because we want to run
+        # pytest from inside the tests/ dir for coverage to work properly (note
+        # this is different than for the notebooks)
+      run: "echo \"script=$(cd tests/ && ls test*py | jq -R -s -c 'split(\"\\n\")[:-1]')\"\
+        \ >> $GITHUB_OUTPUT\n"
   get_notebooks:
     runs-on: ubuntu-latest
     outputs:
       notebook: ${{ steps.get-notebooks.outputs.nb }}
     steps:
-      - uses: actions/checkout@v2
-      - id: get-notebooks
+    - uses: actions/checkout@v3
+    - id: get-notebooks
         # it's weird to me, but the quotes around \n should *not* be escaped or it breaks
```

Comment:
I think this comment is no longer valid.

Reply:
good catch, I removed it